Calculator.c

```diff
@@ -0,0 +1,111 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#include<ctype.h>
+#define MAX 80
+
+int cal(int op1,int op2,char op){
+    if(op=='+'){
+        return op1+op2;
+    }
+    else if(op=='-'){
+        return op1-op2;
+    }
+    else if(op=='*'){
+        return op1*op2;
+    }
+    else if(op=='/'){
+        if(op2==0){
+            printf("Division by Zero error");
+            exit(1);
+        }
+        return op1/op2;
+    }
+    else{    
+        printf("Error:Invalid Operator");
+        exit(1);
+    }
+}
+
+
+int evalvate(char* exp){
```

Comment:
This function can be divided into small functions which do one specific task. Practice writing functions in such a way that one function does one task.